pytorch_lightning/trainer/trainer.py

```diff
@@ -964,10 +964,11 @@ def run_pretrain_routine(self, model: LightningModule):
             ref_model.on_sanity_check_start()
             self.on_sanity_check_start()
 
-            eval_results = self._evaluate(model,
-                                          self.val_dataloaders,
-                                          self.num_sanity_val_steps,
-                                          False)
+            eval_results = self._evaluate(model=model,
+                                          dataloaders=self.val_dataloaders,
+                                          max_batches=[self.num_sanity_val_steps] * len(self.val_dataloaders),
```

Comment:
let's allow to pas also `int` and inside cast the `int` to a `list` of the same element times nb data loaders to do not break back compatibility